src/mixins/itext_behavior.mixin.js

```diff
@@ -564,17 +560,14 @@
       else if (this.__isDraggingOver && !canDrop) {
         //  drop state has changed
         this.__isDraggingOver = false;
-        !this.__isDragging && this.clearContextTop();
       }
-      if (this.__isDraggingOver && canDrop) {
+      if (this.__isDraggingOver) {
```

Comment:
Now I recall this logic
`canDrop` => if another object has accepted the event it has become the drop target and has called `e.preventDefault()` to inform the browser so this object should not try to handle the event. And of course if it `canDrop(e)` is false...

Is there anything I am missing out?

Reply:
needs to be restored IMO

Reply:
i removed this, i m not sure why gitbud doesn't get it

Reply:
i mean there is nothing to revert, the line is white

<img width="541" alt="image" src="https://user-images.githubusercontent.com/1194048/181386017-47527c67-69fe-4563-b42f-7b25e2c96564.png">

Reply:
Ohh

Reply:
OK